com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1427,7 +1427,7 @@ internal void ShutdownInternal()
                 }
             }
 
-            if (IsClient && IsConnectedClient)
+            if (IsClient && IsListening)
```

Comment:
Just curious, was this to handle an edge case scenario where a client had established a connection but had yet to be considered "connected"?

Reply:
Yep, exactly.